src/components/Person/Person.jsx

```diff
@@ -1 +1,19 @@
-// export const Person = ({ person }) => ();
+import React from "react";
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
```

Comment:
code style

Reply:
code style fixed

Reply:
And i forgot about Demo link , please add😊